deploy/install/01-prerequisite/03-crd.yaml

```diff
@@ -2710,6 +2712,7 @@ metadata:
     longhorn-manager: ""
   name: volumes.longhorn.io
 spec:
+  preserveUnknownFields: false
```

Comment:
This was missed. 

I feel we should not have a separate manifest in the manager repo but only keep one place (longhorn/longhorn) instead.